.github/workflows/CI.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         run: cargo build --all-targets --all-features --verbose
 
       - name: Perform unit testing and integration testing
-        run: cargo test --workspace --all-targets --all-features
```

Comment:
@DevQps , I don't think this changes any functionality since there's only one target in bgp-rs. But please correct me if I'm wrong, I don't fully understand this flag